charts/sonarqube-dce/Chart.yaml

```diff
@@ -24,20 +24,8 @@ maintainers:
     email: jeremy.cotineau@sonarsource.com
 annotations:
   artifacthub.io/changes: |
-    - kind: added
-      description: "Add a configurable Prometheus PodMonitor resource"
     - kind: changed
-      description: "Prometheus exporter updated to 0.17.2"
-    - kind: changed
-      description: "Allow setting priorityClassName for StatefulSets"
-    - kind: changed
-      description: "Adds timeoutSeconds parameter to probes"
-    - kind: fixed
-      description: "Fix the volume mounts for the init-fs container"
-    - kind: changed
-      description: "Update SonarQube logo"
-    - kind: changed
-      description: "Bootstrap chart version 8.x.x dedicated to the future SonarQube 10.0"
+      description: "Clarify doc for custom cacert secret"
```

Comment:
Could you please just add your change entry on top of the other changes made in the current release cycle (without deleting them)?

Reply:
I can do this if that's the route desired - my understanding of `artifacthub.io/changes` annotation is that it should capture changes included *only* in that specific chart version. So in this case my patch bump only includes the doc update and should only have that change listed. Happy to be corrected or go along with what is desired here, just wanted to confirm first!

Ref: https://artifacthub.io/docs/topics/annotations/helm/

> This annotation is used to provide some details about the changes introduced by a given chart version.

Reply:
Thanks for sharing and let me clarify this :)

We do a release on ArtifactHUB at end of our release cycle (typically two months). This means that not every version gets released, just the latest available at the end of our release cycle. We want to make sure that this release lists all changes made during the release cycle in the [Changelog view](https://artifacthub.io/packages/helm/sonarqube/sonarqube?modal=changelog). That's why we would need to have your changelog entry added on top of the others. The list will be reset after the next release...

Reply:
Tracking now, this is helpful! I'll get this updated.